src/Shared/Data.php

```diff
@@ -679,14 +691,22 @@ public function isSubscription($orderId)
         return apply_filters($this->pluginId . '_is_subscription_payment', $isSubscription, $orderId);
     }
 
-    public function getAllAvailablePaymentMethods($use_cache = true)
+    public function getAllAvailablePaymentMethods($use_cache = true, $filters = [])
     {
+
         $apiKey = $this->settingsHelper->getApiKey();
         $methods = false;
         $locale = $this->getPaymentLocale();
+        $test_mode = $this->isTestModeEnabled();
         $filters_key = [];
         $filters_key['locale'] = $locale;
+        $filters_key['mode'] = ( $test_mode ? 'test' : 'live' );
+        $filters_key['api'] = 'methods';
         $transient_id = $this->getTransientId(md5(http_build_query($filters_key)));
+        $filters['include'] = 'issuers';
```

Comment:
This line is probably not needed. It is removed with an intersect key.

Reply:
In fact the error was using issuers and not include as key, but after all none of that is needed.